.ci/scripts/clang-format.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+#===============================================================================
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+echo "Using clang-format version: $(clang-format --version)"
+echo "Starting format check..."
+
+for filename in $(find . -type f | grep -P ".*\.(c|cpp|h|hpp|cl)$"); do clang-format -style=file -i $filename; done
```

Comment:
You need to add **.i** extension. The whole basic realization here.

Reply:
will update it  now